docs/pages/access-controls/getting-started.mdx

```diff
@@ -20,35 +20,72 @@ wrap up with creating your own role.
 
 ## Step 1/3. Add local users with preset roles
 
-Teleport provides several preset roles: `editor`, `auditor` and `access`. The
-`editor` role authorizes users to modify cluster configuration, the `auditor`
-role to view audit logs, and `access` role to access cluster resources.
+Teleport provides several preset roles: `editor`, `auditor`, and `access`.
 
+- The `editor` role authorizes users to modify cluster configuration.
+- The `auditor` role authorizes users to view audit logs.
+- The `access` role authorizes users to access cluster resources.
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
+Enterprise contains two additional preset roles: `reviewer` and `requester`.
+
+- The `reviewer` role authorizes users to review application and user group access requests.
```

Comment:
I'm not sure the "application and user group" terminology is something we use in the rest of the docs. Maybe "resources and roles" instead? Also, other parts of the docs capitalize Access Requests.